rosidl_generator_cpp/rosidl_generator_cpp/__init__.py

```diff
@@ -144,6 +144,22 @@ def msg_type_to_cpp(type_):
 
 
 def value_to_cpp(type_, value):
+    """
+    Convert a python value into a string representing that value in C++.
+
+    This is equivalent to primitive_value_to_cpp but can process arrays values as well
+
+    Warning this still processes only primitive types
+    @param type_: a ROS IDL type
+    @type type_: builtin.str
+    @param value: the value to convert
+    @type value: python builtin (bool, int, float, str or list)
+    @returns: a string containing the C++ representation of the value
+    """
+    # :raises: ValueError on if the provided type_ is unknown
+    # """
```

Comment:
Minor nit; it's a little weird to have triple-quotes here and commented-out.  I'd just remove it, as it is syntax.